bom/application/pom.xml

```diff
@@ -69,10 +69,7 @@
         <jakarta.transaction-api.version>1.3.3</jakarta.transaction-api.version>
         <jakarta.validation-api.version>2.0.2</jakarta.validation-api.version>
         <jakarta.websocket-api.version>1.1.2</jakarta.websocket-api.version>
-        <jakarta.xml.bind-api.version>2.3.3</jakarta.xml.bind-api.version>
         <jaxb-runtime.version>2.3.3-b02</jaxb-runtime.version>
-        <!-- just for test dependency convergence -->
-        <jaxb-api.version>2.3.1</jaxb-api.version>
```

Comment:
Not sure about this one and the one above. I think we'd better keep them as there's a good chance it could break dependency convergence.

Reply:
None of them appear to be a dependency of any extension (runtime or deployment).

Reply:
Ok, they are back.

Reply:
Is looking ok now to you @gsmet ?